preprocessing/nextclade/src/loculus_preprocessing/backend.py

```diff
@@ -173,6 +180,44 @@ def submit_processed_sequences(
     logger.info("Processed data submitted successfully")
 
 
+def request_upload(group_id: int, number_of_files: int, config: Config) -> Sequence[FileUploadInfo]:
+    # we need to parse this here, because we don't want the organism in there.
```

Comment:
What is `this`, `here`, and `there`?

Reply:
`backend_host` seems to be a misnomer - if it has organism after the proper hostname, it should be name `backend_url` or something like that not `backend_host` so we might want to fix this now

Reply:
I've changed the comment to 

> we need to parse the backend URL, to extract the API path without the organism component

I agree that `backend_host` isn't the right name, I'd call it `backend_base_url` but it'd be a breaking change because it'd require config changes in the `values.yaml` for the nextclade pipeline, so maybe it's not worth it? I've also added as a comment above the field def: "The backend host is the base API URL including the organism"

Reply:
(marking this as resolved so it's easier to see what's going on, but feel free to un-resolve it again)